### PR TITLE
Update pixel image size

### PR DESCRIPTION
**Pull request by joshdarby**
## Changes

This pull request makes the following changes:

- Updates `$site_icon_url` to get the site icon url to `50`
- Updates the inline `max-width` style for the pixel image to `1em`

## Why

For #69

## Testing/Questions

Features that this PR affects:

- Pixel image size

Questions that need to be answered before merging:

- [ ] Is this PR targeting the correct branch in this repository?

Steps to test this PR:

1. Copy the shareable content from one post
2. Paste copied content into a new post
3. Verify the pixel image in the content is a better size than before

### CODE REVIEW

File: includes/shareable-content.php [hunk 1]
```diff
@@ -88,7 +88,7 @@
  */
 $pixel = sprintf(
 	// %1$s is the javascript source, %2$s is the post ID, %3$s is the plugins URL
-	'<img id="republication-tracker-tool-source" src="%1$s/?republication-pixel=true&post=%2$s&ga=%3$s" style="max-width:200px;">',
+	'<img id="republication-tracker-tool-source" src="%1$s/?republication-pixel=true&post=%2$s&ga=%3$s" style="max-width:1em;">',
```
Comment (benlk) [inline includes/shareable-content.php]: Should this be max-width or max-height?

Comment (joshdarby) [reply includes/shareable-content.php]: I think either way makes sense.

Comment (benlk) [reply includes/shareable-content.php]: If the image we're grabbing is square, that's true.

**Review by adekbadek — APPROVED** [review]

**Comment by adekbadek** [comment]
Oh wait it's [already a 1x1 pixel](https://github.com/Automattic/republication-tracker-tool/blob/master/includes/pixel.php#L114-L115). Closing.

**Pull request closed without merging**